.golangci.yaml

```diff
@@ -57,7 +57,7 @@ linters-settings:
       limitations under the License.
 issues:
   fix: true
-  exclude: ['declaration of "(err|ctx)" shadows declaration at']
+  exclude: ['declaration of "(err|ctx)" shadows declaration at', 'getInstanceAttribute` - `attribute` always receives `"userData"']
```

Comment:
CI tests are failing because there is a method which has userdata as the arg at all time. We can either exclude the lint check or otherwise refactor the method so it does not always receive the same argument.